java/src/org/openqa/selenium/devtools/CdpClientGenerator.java

```diff
@@ -111,8 +111,8 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
   }
 
   private static class Model {
-    private List<Domain> domains = new ArrayList<>();
-    private String basePackage;
+    private final List<Domain> domains = new ArrayList<>();
```

Comment:
It might be a good idea to avoid changes to this file. It is used to generate CDP version-specific APIs. There is no way to test this in our tests.

Reply:
Generally I agree, but in this case it's impossible to break this code. If it compiles - it works. It was a private field, and it stays a private field.